driver/config/map.json

```diff
@@ -39,6 +39,10 @@
         "filename": "case10.py",
         "isLive": false
     },
+    "11": {
+        "modulename": "case11",
+        "filename": "case11.py"
+    },
```

Comment:
Please merge master into your branch again, this change is a leftover from your PR before